packages/open-ui/src/components/Image/Image.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { MarginStyleKeys } from 'interfaces/MarginStyleKeys';
+import { BorderRadiusOption } from 'interfaces/BorderRadiusOption';
+import { getMarginProps } from 'utils/helpers';
+import styles from './Image.module.scss';
+import { toSentenceCase } from 'utils/strings/to-sentence-case';
+
+interface ImageProps {
```

Comment:
@remotealex should we be using interface here ? Types seems relevant aint ? or where should we use Types ?

Reply:
I think CRA warns to use interfaces when they are _object_ shaped. Types seem to be for _value_ based types.

Reply:
Oh yeah! and you can `extend` interfaces which is handy for this sometimes

Reply:
value based meaning like `type = 'somevalue' | 'somevalue2'` ?

Reply:
Oh yeah! and you can `extend` interfaces which is handy for this sometimes